azure-kusto-ingest/tests/test_ingestion_blob_info.py

```diff
@@ -19,6 +18,8 @@
     ValidationImplications,
 )
 
+from azure.kusto.data.data_format import DataFormat
```

Comment:
External imports should be put before internal imports (https://www.python.org/dev/peps/pep-0008/#imports)